source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,97 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+You can remove any unused index that isn't a :ref:`unique index
+<unique-indexes>`. 
+
+The following sections show the three ways to remove indexes:
+
+- Using an index specification document
+- Using an indexed name field
+- Removing all indexes
```

Comment:
Nitpick: this item does not specify a "way to remove indexes" like the other items in this list. Instead, consider adding that info to the beginning:

```suggestion
- Using a wildcard character to remove all indexes
```